docs/Using-Fleet/Detail-Queries-Summary.md

```diff
@@ -731,4 +731,5 @@ SELECT date, title FROM windows_update_history WHERE result_code = 'Succeeded'
 
 
 
-<meta name="pageOrderInSection" value="1600">
\ No newline at end of file
+<meta name="pageOrderInSection" value="1600">
+<meta name="description" value="Learn about Fleet's hardcoded queries used to populate device details.">
```

Comment:
I don't think this can be edited here in any kind of way that sticks; this file is auto-generated (I remember this one because I have often wanted to rename to "Host vitals" and realized, dangit, I can't right now!!)

[Looks straightforward enough to edit](https://github.com/fleetdm/fleet/blob/main/server/service/osquery_utils/gen_queries_doc.go) but might want to do that as a separate PR since it has a different codeowner and you'll probably get a quicker approval with a smaller change 😅

Reply:
Thanks for the clarification! I will make a separate PR for this file.